src/saving_tools.jl

```diff
@@ -167,7 +168,8 @@ the project's gitpath). Do nothing if a key `gitcommit` already exists
 repository is not found. If the git repository is dirty, i.e. there
 are un-commited changes, and `storepatch` is true, then the output of `git diff HEAD` is stored
 in the field `gitpatch`.  Note that patches for binary files are not
-stored. You can use [`isdirty`](@ref) to check if a repo is dirty.
+stored. You can use [`isdirty`](@ref) to check if a repo is dirty. If the `commit message` is set to `true`, 
```

Comment:
```suggestion
stored. You can use [`isdirty`](@ref) to check if a repo is dirty. 
If the keyword `commit_message` is set to `true`, 
```

it is important to be _precise_ in documentation strings.